keyboards/studiokestra/nascent/info.json

```
@@ -0,0 +1,80 @@
+{
+    "keyboard_name": "Nascent",
+    "url": "https://studiokestra.ca/",
+    "maintainer": "Studio Kestra",
```

```suggestion
    "keyboard_name": "Nascent",
    "manufacturer": "Studio Kestra",
    "url": "https://studiokestra.ca/",
    "maintainer": "Studio Kestra",
    "usb": {
        "vid": "0x7C10",
        "pid": "0x0165",
        "device_version": "0.0.1"
    },
```
I haven't actually done the S boards yet, so this is fine, but might as well do it here.

---

Sort order in my head was different, sorry!